netutils/utils_linux.go

```diff
@@ -0,0 +1,29 @@
+// +build linux
```

Comment:
Now that windows is going to be supported for the daemon and therefore for libnetwork as well, we need a utils_unsupported to atleast to satisfy a build level windows CI task